pkg/sqlfs/hive_writer_test.go

```diff
@@ -35,7 +36,7 @@ func hasHDFSDir(hdfsPath string) bool {
 }
 
 func TestNewHiveWriter(t *testing.T) {
-	testDriver = getEnv("SQLFLOW_TEST_DB", "hive")
+	testDriver := os.Getenv("SQLFLOW_TEST_DB")
```

Comment:
This quick fix is off the topic of this PR. I have to add it because I need it to fix the failure of the unit test. Without this fix, when I run `go test -v ./...`, the call to `getEnv` returns "hive" as the default value, which then triggers the run of `TestNewHiveWriter` even if there is no Hive Server setup, and the unit test fails.